<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRadarComponent</class>
 <widget class="QFrame" name="CRadarComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radar screen</string>
  </property>
  <layout class="QVBoxLayout" name="vl_RadarComponent">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="BlackGui::Views::CRadarView" name="gv_RadarView"/>
   </item>
   <item>
    <widget class="QFrame" name="fr_RadarSettings">
     <layout class="QGridLayout" name="gl_RadarSettings">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item row="0" column="1">
       <widget class="QCheckBox" name="cb_GroundSpeed">
        <property name="toolTip">
         <string>Ground speed</string>
        </property>
        <property name="text">
         <string>GS</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="cb_Grid">
        <property name="text">
         <string>Grid</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="cb_Heading">
        <property name="text">
         <string>Track line</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cb_LockNorth">
        <property name="text">
         <string>Lock North</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_FontSize">
        <property name="text">
         <string>Font size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="cb_Altitude">
        <property name="toolTip">
         <string>Altitude</string>
        </property>
        <property name="text">
         <string>ALT</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cb_Callsign">
        <property name="text">
         <string>Callsign</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lbl_Range">
        <property name="text">
         <string>Range</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QSpinBox" name="sb_FontSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cb_RadarRange"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Views::CRadarView</class>
   <extends>QGraphicsView</extends>
   <header>blackgui/views/radarview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>gv_RadarView</tabstop>
  <tabstop>cb_Callsign</tabstop>
  <tabstop>cb_GroundSpeed</tabstop>
  <tabstop>cb_Altitude</tabstop>
  <tabstop>cb_LockNorth</tabstop>
  <tabstop>cb_Heading</tabstop>
  <tabstop>cb_Grid</tabstop>
  <tabstop>sb_FontSize</tabstop>
  <tabstop>cb_RadarRange</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
